src/modules/flow/http-client/http-client.c

```diff
@@ -399,8 +422,8 @@ common_get_process(struct sol_flow_node *node, void *data, uint16_t port, uint16
     }
 
     sol_http_params_init(&params);
-    if (!sol_http_param_add(&params,
-        SOL_HTTP_REQUEST_PARAM_HEADER("Accept", "application/json"))) {
+    if (mdata->accept && !sol_http_param_add(&params,
+        SOL_HTTP_REQUEST_PARAM_HEADER("Accept", mdata->accept))) {
```

Comment:
`mdata->accept` is an option optional. This will always fail if this option was not given.
```
if (mdata->accept) {
    if(!sol_httt_param_add(...)) {
        ...
    }
}
```